### ISSUE TITLE
mapApp() looks different (firefox vs safari)

### ISSUE THREAD

**Issue by dankelley** [ArgoCanada/argoFloats#235]
Amusingly, firefox gets one of the navigation arrows wrong.

I do not think this is a bug in the app, and so I am going to close this issue after a day or two.  My only purpose in posting it is to let other argoFloats people know of the problem.  (Maybe some could test it on other machines??)

I think I have the symbols right.  The code to make these arrows is as below; note that they are similar unicode mappings, in sequence.
```R
actionButton("goS", "\U21E9"),
actionButton("goN", "\U21E7"),
actionButton("goW", "\U21E6"),
actionButton("goE", "\U21E8"),
```

Snapshots (of the ace-net site, which is *not* for public use, so I will not cite it here) are below.

**Safari/macos**

This is what I wanted. And a firefox/linux user found the same, I think.

<img width="273" alt="Screen Shot 2020-09-09 at 8 14 59 AM" src="https://user-images.githubusercontent.com/99469/92591653-af560600-f274-11ea-85fd-d7f85f8990b7.png">

**Firefox/macos**

Not what I wanted!

<img width="287" alt="Screen Shot 2020-09-09 at 8 14 51 AM" src="https://user-images.githubusercontent.com/99469/92591655-afee9c80-f274-11ea-8298-6316a86eac33.png">

**Comment by dankelley** [comment]
Q to @j-harbin -- do you think I should put in letters "S", "N" etc, instead of the arrows?  

Personally, I prefer the arrows, and I do not want to change to letters.  (I guess if S,N,E,W are together, a person might twig to what they mean.). Basically, I don't see why what seems to be a firefox deficiency should affect argoFloats users in this way.  (Also, screenshots from a linux user suggest the arrows came out ok with firefox there.)

**Comment by dankelley** [comment]
Oh, to heck with it.  I want this working for class today and it's a 1/2-hour installation time so I went ahead and changed to UTF symbols (as opposed to unicode symbols).  It now looks as below.  I can live with this, so I'll close this issue.

PS. I tried installing icons but I just wasn't sure where to place them so it would work for both in-R use and web use.

<img width="250" alt="Screen Shot 2020-09-09 at 9 20 15 AM" src="https://user-images.githubusercontent.com/99469/92597476-d1a05180-f27d-11ea-844a-af6b0f8e9ba4.png">
